en/docs/using-wso2-identity-server/invoke-the-oauth-introspection-endpoint.md

```diff
@@ -87,7 +71,7 @@ invoke the OAuth introspection endpoint for the super tenant users.
         `           <IS_HOME>/repository/conf/identity/identity.xml          `
         file.
     
-        ``` xml
+        ``` toml
             <Resource context="(.*)/oauth2/introspect(.*)" secured="true" http-method="all">
```

Comment:
Is this the toml config?

Reply:
No. This property is still untemplated. Created this git issue [1] 

[1] https://github.com/wso2/product-is/issues/6132